src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -116,13 +116,13 @@ interface Counts {
 // and `BaseFilterContext<AuctionResultFilters>`.
 export interface ArtworkFilterContextProps {
   /** The current artwork filter state (which determines the network request and the url querystring) */
-  filters?: ArtworkFilters
+  filters?: ArtworkFiltersState
```

Comment:
I hope this isn't controversial.

If we want `ArtworkFilterContext` consumers to be able to see when the filter state has been `reset` in a type-safe way, we need to pass this down as `ArtworkFiltersState` instead of `ArtworkFilters`, since [that is the interface that knows](https://github.com/artsy/force/blob/da51f4687d49cf16810afac3e3b1b30df7817d61/src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx#L71) about the `reset` boolean.